src/hyperion/device_setup_plans/setup_panda.py

```diff
@@ -67,44 +77,65 @@ def get_seq_table(
 
     exposure_distance_x_counts = int(exposure_distance_mm * MM_TO_ENCODER_COUNTS)
 
+    num_pulses = parameters.x_steps
+
+    delay_between_pulses = time_between_steps_ms * TICKS_PER_MS
+
+    PULSE_WIDTH_US = 1
```

Comment:
I believe we don't actually need a pulse block. The SEQ.OUT.A can be treated as our pulse, with width determined by `time1` and period determined by `time2`